index.js

```diff
@@ -91,6 +91,69 @@ say.speak = function(text, voice, speed, callback) {
   });
 };
 
+say.export = function(text, output, params, callback) {
+  var commands, pipedData;
+
+  if (!text) {
+    // throw TypeError because API was used incorrectly
+    throw new TypeError('Must provide text parameter');
+  }
+
+  if (!output || typeof output !== 'string') {
+    // throw TypeError because API was used incorrectly
+    throw new TypeError('Must provide output parameter');
+  }
+
+  if (typeof params !== 'object'){
+    params = {voice: 'Alex', speed: 175};
+  } else {
+    params.voice = params.voice || 'Alex';
+    params.speed = params.speed || 175;
+  }
+
+  if (typeof callback !== 'function') {
+    callback = function() {};
+  }
+
+  // tailor command arguments to specific platforms
+  if (process.platform === 'darwin') {
+    commands = ['-v', params.voice, text];
+
+    commands.push('-r', params.speed, '-o', output, '--data-format=LEF32@32000');
```

Comment:
When the voice and speed params are missing, simply don't push them to the commands array.